<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <widget class="QWidget" name="centralwidget"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="separator"/>
    <addaction name="actionClose"/>
    <addaction name="actionClose_All"/>
    <addaction name="separator"/>
    <addaction name="actionReload"/>
    <addaction name="separator"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_As"/>
    <addaction name="actionSave_All"/>
    <addaction name="separator"/>
    <addaction name="actionPrint"/>
    <addaction name="actionPrint_Preview"/>
    <addaction name="actionPrint_Setup"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionUndo"/>
    <addaction name="actionRedo"/>
    <addaction name="separator"/>
    <addaction name="actionCut"/>
    <addaction name="actionCopy"/>
    <addaction name="actionPaste"/>
    <addaction name="separator"/>
    <addaction name="actionInsert_Date"/>
    <addaction name="separator"/>
    <addaction name="actionColumn_Mode"/>
   </widget>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>Settings</string>
    </property>
    <addaction name="actionColors"/>
    <addaction name="actionFonts"/>
    <addaction name="actionTab_Size"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionDiamond_Help"/>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuSearch">
    <property name="title">
     <string>Search</string>
    </property>
    <addaction name="actionFind"/>
    <addaction name="actionReplace"/>
    <addaction name="separator"/>
    <addaction name="actionFind_Next"/>
    <addaction name="actionFind_Prev"/>
    <addaction name="separator"/>
    <addaction name="actionAdvanced_Find"/>
    <addaction name="separator"/>
    <addaction name="actionGo_Line"/>
    <addaction name="actionGo_Column"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionLine_Numbers"/>
    <addaction name="separator"/>
    <addaction name="actionShow_Spaces"/>
    <addaction name="actionShow_Spaces_EOL"/>
    <addaction name="actionShow_Tabs"/>
    <addaction name="actionShow_Line_Break"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menuDocument">
    <property name="title">
     <string>Document</string>
    </property>
    <widget class="QMenu" name="menuFile_Format">
     <property name="title">
      <string>File Format</string>
     </property>
     <addaction name="actionDOS_CR_LF"/>
     <addaction name="actionUnix_Format_LF_Only"/>
     <addaction name="actionMac_Format_CR_Only"/>
    </widget>
    <widget class="QMenu" name="menuSyntax_Type">
     <property name="title">
      <string>Syntax Type</string>
     </property>
     <addaction name="actionDefault"/>
     <addaction name="actionPlain_Text"/>
     <addaction name="separator"/>
     <addaction name="actionC_C"/>
     <addaction name="actionCSS"/>
     <addaction name="actionDoxygen"/>
     <addaction name="actionHTML"/>
     <addaction name="actionJava"/>
     <addaction name="actionJava_Script"/>
     <addaction name="actionMakefile"/>
     <addaction name="actionShell_Script"/>
     <addaction name="actionPerl_Script"/>
    </widget>
    <widget class="QMenu" name="menuTabs_Spaces">
     <property name="title">
      <string>Tabs &amp;&amp; Spaces</string>
     </property>
     <addaction name="actionConvert_Tabs_to_Spaces_2"/>
     <addaction name="actionConvert_Spaces_to_Tabs"/>
     <addaction name="actionRemove_Trailing_Spaces"/>
    </widget>
    <addaction name="menuSyntax_Type"/>
    <addaction name="menuFile_Format"/>
    <addaction name="separator"/>
    <addaction name="menuTabs_Spaces"/>
   </widget>
   <widget class="QMenu" name="menuWindow">
    <property name="title">
     <string>Window</string>
    </property>
    <addaction name="actionNew_Tab"/>
    <addaction name="actionClose_Tab"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Tools</string>
    </property>
    <widget class="QMenu" name="menuMacros">
     <property name="title">
      <string>Macros</string>
     </property>
     <addaction name="actionBegin_Recording"/>
     <addaction name="actionStop_Recording"/>
     <addaction name="separator"/>
     <addaction name="actionPlay_Macro"/>
    </widget>
    <addaction name="menuMacros"/>
    <addaction name="actionSpell_Check"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuSearch"/>
   <addaction name="menuView"/>
   <addaction name="menuDocument"/>
   <addaction name="menuTools"/>
   <addaction name="menuSettings"/>
   <addaction name="menuWindow"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionNew">
   <property name="icon">
     <iconset resource="diamond.qrc">
     <normaloff>:/resources/new.png</normaloff>:/resources/new.png</iconset>
   </property>
   <property name="text">
    <string>New</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Open...</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save </string>
   </property>
  </action>
  <action name="actionSave_As">
   <property name="text">
    <string>Save As...</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="text">
    <string>Close</string>
   </property>
  </action>
  <action name="actionCut">
   <property name="text">
    <string>Cut</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="text">
    <string>Copy</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="text">
    <string>Paste</string>
   </property>
  </action>
  <action name="actionColors">
   <property name="text">
    <string>Colors</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionDiamond_Help">
   <property name="text">
    <string>Help Topics...</string>
   </property>
  </action>
  <action name="actionNew_Tab">
   <property name="text">
    <string>New Tab</string>
   </property>
  </action>
  <action name="actionBegin_Recording">
   <property name="text">
    <string>Begin Recording</string>
   </property>
  </action>
  <action name="actionStop_Recording">
   <property name="text">
    <string>End Recording</string>
   </property>
  </action>
  <action name="actionPlay_Macro">
   <property name="text">
    <string>Play Macro</string>
   </property>
  </action>
  <action name="actionFonts">
   <property name="text">
    <string>Fonts</string>
   </property>
  </action>
  <action name="actionDOS_CR_LF">
   <property name="text">
    <string>DOS Format (CR/LF)</string>
   </property>
  </action>
  <action name="actionUnix_Format_LF_Only">
   <property name="text">
    <string>Unix Format (LF Only)</string>
   </property>
  </action>
  <action name="actionMac_Format_CR_Only">
   <property name="text">
    <string>Mac Format (CR Only)</string>
   </property>
  </action>
  <action name="actionC_C">
   <property name="text">
    <string>C/C++</string>
   </property>
  </action>
  <action name="actionPerl_Script">
   <property name="text">
    <string>Perl Script</string>
   </property>
  </action>
  <action name="actionCSS">
   <property name="text">
    <string>CSS</string>
   </property>
  </action>
  <action name="actionPlain_Text">
   <property name="text">
    <string>Plain Text</string>
   </property>
  </action>
  <action name="actionDefault">
   <property name="text">
    <string>Default</string>
   </property>
  </action>
  <action name="actionHTML">
   <property name="text">
    <string>HTML</string>
   </property>
  </action>
  <action name="actionDoxygen">
   <property name="text">
    <string>Doxygen</string>
   </property>
  </action>
  <action name="actionJava">
   <property name="text">
    <string>Java </string>
   </property>
  </action>
  <action name="actionMakefile">
   <property name="text">
    <string>Makefile</string>
   </property>
  </action>
  <action name="actionShell_Script">
   <property name="text">
    <string>Shell Script</string>
   </property>
  </action>
  <action name="actionJava_Script">
   <property name="text">
    <string>Javascript</string>
   </property>
  </action>
  <action name="actionClose_All">
   <property name="text">
    <string>Close All</string>
   </property>
  </action>
  <action name="actionReload">
   <property name="text">
    <string>Reload</string>
   </property>
  </action>
  <action name="actionSave_All">
   <property name="text">
    <string>Save All</string>
   </property>
  </action>
  <action name="actionPrint">
   <property name="text">
    <string>Print</string>
   </property>
  </action>
  <action name="actionPrint_Preview">
   <property name="text">
    <string>Print Preview</string>
   </property>
  </action>
  <action name="actionPrint_Setup">
   <property name="text">
    <string>Print Setup</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="text">
    <string>Undo</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="text">
    <string>Redo</string>
   </property>
  </action>
  <action name="actionInsert_Date">
   <property name="text">
    <string>Insert Date</string>
   </property>
  </action>
  <action name="actionColumn_Mode">
   <property name="text">
    <string>Column Mode</string>
   </property>
  </action>
  <action name="actionFind">
   <property name="text">
    <string>Find...</string>
   </property>
  </action>
  <action name="actionReplace">
   <property name="text">
    <string>Replace...</string>
   </property>
  </action>
  <action name="actionFind_Next">
   <property name="text">
    <string>Find Next</string>
   </property>
  </action>
  <action name="actionFind_Prev">
   <property name="text">
    <string>Find Prev</string>
   </property>
  </action>
  <action name="actionAdvanced_Find">
   <property name="text">
    <string>Advanced Find...</string>
   </property>
  </action>
  <action name="actionGo_Line">
   <property name="text">
    <string>Go to Line</string>
   </property>
  </action>
  <action name="actionGo_Column">
   <property name="text">
    <string>Go to Column</string>
   </property>
  </action>
  <action name="actionLine_Numbers">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Line Numbers</string>
   </property>
  </action>
  <action name="actionTab_Size">
   <property name="text">
    <string>Tab Size</string>
   </property>
  </action>
  <action name="actionSpell_Check">
   <property name="text">
    <string>Spell Check</string>
   </property>
  </action>
  <action name="actionShow_Tabs">
   <property name="text">
    <string>Show Tabs</string>
   </property>
  </action>
  <action name="actionShow_Line_Break">
   <property name="text">
    <string>Show Line Break</string>
   </property>
  </action>
  <action name="actionShow_Spaces">
   <property name="text">
    <string>Show Spaces</string>
   </property>
  </action>
  <action name="actionShow_Spaces_EOL">
   <property name="text">
    <string>Show Spaces EOL</string>
   </property>
  </action>
  <action name="actionConvert_Tabs_to_Spaces_2">
   <property name="text">
    <string>Convert Tabs to Spaces</string>
   </property>
  </action>
  <action name="actionConvert_Spaces_to_Tabs">
   <property name="text">
    <string>Convert Spaces to Tabs</string>
   </property>
  </action>
  <action name="actionRemove_Trailing_Spaces">
   <property name="text">
    <string>Remove Trailing Spaces</string>
   </property>
  </action>
  <action name="actionClose_Tab">
   <property name="text">
    <string>Close Tab</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
